internal/command/checks/command.go

```diff
@@ -0,0 +1,106 @@
+package checks
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.stackrox.io/kube-linter/internal/builtinchecks"
+	"golang.stackrox.io/kube-linter/internal/check"
+	"golang.stackrox.io/kube-linter/internal/command/common"
+	"golang.stackrox.io/kube-linter/internal/defaultchecks"
+)
+
+var (
+	dashes = func() string {
+		var sb strings.Builder
+		for i := 0; i < 30; i++ {
+			sb.WriteRune('-')
+		}
+		return sb.String()
+	}()
+
+	formatsToRenderFuncs = map[string]func([]check.Check, io.Writer) error{
+		common.PlainFormat:    renderPlain,
+		common.MarkdownFormat: renderMarkdown,
+	}
+)
+
+func renderPlain(checks []check.Check, out io.Writer) error { //nolint:unparam // The function signature is required to match formatToRenderFuncs
```

Comment:
Why don't you use a Go template for this?

Reply:
This function is pretty simple, so I think it's better to keep it as is. I find normal code easier to read (plus the compile-time safety is nice).